data_splitting.py

```diff
@@ -121,8 +122,10 @@ def process_data_splitting_scenario(scenario):
         else:
             raise NameError('This testset_option [' + scenario.testset_option + '] scenario is not recognized')
             
-        node = Node(x_node_train, x_node_test, y_node_train, y_node_test)
+        node = Node(x_node_train, x_node_test, y_node_train, y_node_test, 
+                    str(nodeId))
```

Comment:
Are you sure @celinejacques that it has to be stringified here? Looks like in the rest of the code you keep it an integer and stringify it each time you print it.

Reply:
My idea was that the name will be a string such as the name of the center. 
Which option do you prefer: 
- I change node number to string before initialize it (so nodeId or node_id now is a string and not stringify it here)
- I left it as a number and stringify it only for print. 
- Other idea? 
I will change the prints when it is needed with the option you prefer.